src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -226,6 +226,9 @@ protected function describeContainerServices(ContainerBuilder $builder, array $o
         $rawOutput = isset($options['raw_text']) && $options['raw_text'];
         foreach ($this->sortServiceIds($serviceIds) as $serviceId) {
             $definition = $this->resolveServiceDefinition($builder, $serviceId);
+            if ('\\' === substr($serviceId, -1)) {
+                $serviceId = substr($serviceId, 0, -1);
```

Comment:
But this means that the displayed service ID will be wrong, doesn't it?

Reply:
Yeah, you are right, we should escape without removing it

Reply:
I think you need to take a look at the `OutputFormatter::escape()` method

Reply:
Thanks @xabbuh that's what I needed !